signalfx-org-metrics/metrics.yaml

```diff
@@ -479,15 +479,17 @@ sf.org.num.gcpStackdriverClientCallCountThrottles:
   metric_type: counter
   title: sf.org.num.gcpStackdriverClientCallCountThrottles
 
-sf.org.num.metric:
-  brief: Number of unique metrics across all MTS
-  description: 'Number of unique metrics across all MTS.
+sf.org.num.LimitedMetricTimeSeriesCreateCallsByCategoryType:
+  brief: Number of metric time series created that are over the limit for an MTS category
+  description: 'Number of metric time series created that are over the purchased
+    limit for that MTS category (such as Host, Container, Custom, Tracing and Bundled).
 
     Dimension(s): `orgId`
 
     Data Resolution: 15 minutes'
   metric_type: gauge
-  title: sf.org.num.metric
```

Comment:
Undo this remove?

Reply:
That was title from another entry I'd copied over as a syntax model for the new metric.